docs/rrsets.rst

```diff
@@ -198,11 +198,32 @@ command::
     curl -X GET https://desec.io/api/v1/domains/:name/rrsets/ \
         --header "Authorization: Token {token}"
 
-to retrieve the contents of a zone that you own.
+to retrieve the contents of a zone that you own.  RRsets are returned in
+reverse chronological order of their creation.
```

Comment:
Same as above

Reply:
This is a new side-effect feature: The paginator paginates based on the created timestamp.